torch/_dynamo/variables/torch.py

```diff
@@ -918,19 +918,25 @@ def speculate_subgraph(f, sub_args, graph_checkpoint, checkpoint):
                     args = []
                     # One argument to graph per sub_args
                     for a in sub_args:
-                        if isinstance(a, TensorVariable):
+                        if isinstance(a, ConstantVariable):
+                            proxy = tracer.create_graph_input("const")
```

Comment:
For my understanding, how does one obtain a ConstantVariable?

Reply:
As in, where do they come from? Or mechanically?

Mechanically:

`ConstantVariable(foo)` will do it - foo can be whatever basically. 

They do not have a proxy, though I've thought about maybe doing something akin to ProxiedConstant for when we flip between Constant/SymNode (SymNodeVariable is the closest we have to something like that)

Where they come from:

Seeing a constant during symbolic evaluation.